src/app/(main)/my-voyage/[teamId]/sprints/components/sections/Notes.tsx

```diff
@@ -41,15 +92,17 @@ export default function Notes() {
         {...register("notes")}
         errorMessage={errors.notes?.message}
       />
-      <Button
-        type="submit"
-        variant="outline"
-        size="md"
-        className="self-center"
-        disabled={!isDirty || !isValid}
-      >
-        Save
-      </Button>
+      {isDirty && (
+        <Button
+          type="submit"
+          variant="outline"
+          size="md"
+          className="self-center"
+          disabled={!isDirty || !isValid || editMeetingLoading}
+        >
+          Save
+        </Button>
+      )}
```

Comment:
Ok so, playing around with this I found something interesting. I got rid of the isDirty conditional here and when I first open the section, it doesn't recognize the isdirty state properly and the button isn't disabled. However, if I add the default value back in the RHF config, it seems to work properly. I think I had something similar in ideation form, except I reset the values in a useffect. I think it accomplishes the same thing, but I'm not sure.

Either way, I think I would prefer having this disabled button instead of hiding the button and rendering it when user types and keeping that default values thing seems to do the trick.

I would also like to have the loading state (spinner) on the button when user saves

Reply:
1. I've decided to use `useEffect` to reset values like in all other forms. Forgot about `useEffect` entirely 🙃
2. Added the spinner
3. The button is visible now all the time